pallets/mint_with_fees/src/lib.rs

```diff
@@ -0,0 +1,124 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+// pub use pallet::*;
+
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod tests;
+
+pub use pallet::*;
+pub use frame_support::{
+        dispatch::DispatchResult, pallet_prelude::*,
+        traits::Currency,
+        sp_runtime::traits::{CheckedConversion, CheckedDiv}
+    };
+pub use frame_system::{
+        pallet_prelude::*,
```

Comment:
`frame_support` and `frame_system` `pallet_prelude::*` goes in the `mod pallet`
https://docs.substrate.io/rustdocs/latest/src/pallet_treasury/lib.rs.html#136-136